examples/v2/chat2.nim

```diff
@@ -31,7 +31,9 @@ const Help = """
 """
 
 const DefaultTopic = "waku"
-const DefaultContentTopic = "dingpu"
+
+const Dingpu = "dingpu".toBytes
+const DefaultContentTopic = ContentTopic(uint32.fromBytes([Dingpu[0], Dingpu[1], Dingpu[2], Dingpu[3]]))
```

Comment:
This seems weird and manual to me. Why not do something like `cast[seq[byte](...)`?

Reply:
This currently seems like the most decent way, the fromBytes function needs a fixed sized array that is the length of the int.

Reply:
You should be able to pass just `Dingpu`, as long as it contains 4 bytes that should work. Or you can take the 0..4 slice to be explicit I guess.